### PR TITLE
Fix ScrapeTool parameter validation errors

### PR DESCRIPTION
**Pull request by likenamehaojie**
Changed placeholders to I18nObject instances to pass validation checks.

# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [ ] Please open an issue before creating a PR or link to an existing issue
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description
Fix validation error for ScrapeTool parameters

The previous implementation contained invalid placeholders for the
ScrapeTool parameters, causing validation errors during runtime.
This was due to the placeholders not being instances of I18nObject.

To resolve this, I've updated the placeholders to use I18nObject instances,
providing correct translations for 'zh_Hans', 'pt_BR', and 'en_US'.
This ensures that the tool's parameters are now properly validated.

Related issue: #123
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency upgrade

### CODE REVIEW

File: api/pyproject.toml
Comment (crazywoola) [inline api/pyproject.toml:1]: This is unrelated to this pr, please remain this file and poetry.lock file untouched.

**Review by crazywoola — CHANGES_REQUESTED** [review]

File: api/pyproject.toml [hunk 1]
```diff
@@ -250,3 +250,7 @@ optional = true
 [tool.poetry.group.lint.dependencies]
 ruff = "~0.5.1"
 dotenv-linter = "~0.5.0"
+
+[[tool.poetry.source]]
```
Comment (laipz8200) [inline api/pyproject.toml:254]: Please do not add a third-party source.

File: api/core/tools/provider/builtin/spider/tools/scraper_crawler.yaml [hunk 2]
```diff
@@ -74,7 +74,6 @@ parameters:
     human_description:
       en_US: blacklist a set of paths that you do not want to crawl. you can use regex patterns to help with the list.
       zh_Hans: 指定一组不想爬取的路径。您可以使用正则表达式模式来帮助定义列表。
-    placeholder: /blog/*, /about
```
Comment (laipz8200) [inline api/core/tools/provider/builtin/spider/tools/scraper_crawler.yaml:77]: You can leave this placeholder, but please use the same i18n format as the label.

File: api/core/tools/provider/builtin/spider/tools/scraper_crawler.yaml [hunk 3]
```diff
@@ -85,7 +84,6 @@ parameters:
     human_description:
       en_US: Whitelist a set of paths that you want to crawl, ignoring all other routes that do not match the patterns. You can use regex patterns to help with the list.
       zh_Hans: 指定一组要爬取的路径，忽略所有不匹配模式的其他路由。您可以使用正则表达式模式来帮助定义列表。
-    placeholder: /blog/*, /about
```
Comment (laipz8200) [inline api/core/tools/provider/builtin/spider/tools/scraper_crawler.yaml:88]: ditto

**Review by laipz8200 — CHANGES_REQUESTED** [review]

File: .vscode/launch.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
             "name": "Python: Flask",
             "type": "debugpy",
             "request": "launch",
-            "python": "${workspaceFolder}/api/.venv/bin/python",
```
Comment (laipz8200) [inline .vscode/launch.json:8]: Please refrain from modifying our config file.

File: .vscode/launch.json [hunk 5]
```diff
@@ -25,7 +25,7 @@
             "name": "Python: Celery",
             "type": "debugpy",
             "request": "launch",
-            "python": "${workspaceFolder}/api/.venv/bin/python",
```
Comment (laipz8200) [inline .vscode/launch.json:28]: ditto

**Comment by likenamehaojie** [comment]
我就改了一行代码，怎么多出这么多？watercrawl不能使用

**Pull request closed without merging**